jax/experimental/jax2tf/examples/tf_js/quickdraw/quickdraw.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from absl import app # type: ignore
+import os # type: ignore
+import time
+from typing import Callable
+
+import flax # type: ignore
+from flax import linen as nn
+from flax.training import common_utils # type: ignore
+
+import jax # type: ignore
+from jax import lax
+from jax import numpy as jnp
+from jax.experimental import jax2tf # type: ignore
+jax2tf.jax2tf.ENABLE_TF_CONVOLUTION = True
+
+from jax.experimental.jax2tf.examples.saved_model_lib import save_model # type: ignore
```

Comment:
I suggest importing modules, not functions:
```
from jax.experimental.jax2tf.examples impot saved_model_lib
```

Then when we see `saved_model_lib.save_model` we know right away what is being called.